src/NServiceBus.Hosting.Windows/StartableAndStoppable/StartableAndStoppableRunner.cs

```diff
@@ -96,8 +109,30 @@ public Task Start(IMessageSession session)
             }
         }
 
+        CancellationTokenSource LongRunningWarning(string message)
+        {
+            var tokenSource = new CancellationTokenSource();
+
+            Task.Run(() =>
+            {
+                try
+                {
+                    Task.Delay(LongRunningWarningTimeSpan).Wait(tokenSource.Token);
```

Comment:
Pretty sure the reason here is https://github.com/Particular/NServiceBus.Host/pull/72/files/50c03956d61cdc85c5d0ad3bbe9f26f4a1ebcd98#diff-bdc6a71a7cbb3fc456dc53ca873c90a4R77

but I'll "await" (pun intended) the async master @danielmarbach 's answer